### PR TITLE
import: add '--file' option

### PR DESCRIPTION
**Pull request by DiPaolo**
> You may disregard these recommendations if you used the **Edit on GitHub** button from dvc.org to improve a doc in place.

❗ Please read the guidelines in the [Contributing to the Documentation](https://dvc.org/doc/user-guide/contributing/docs) list if you make any substantial changes to the documentation or JS engine.

🐛 Please make sure to mention `Fix #issue` (if applicable) in the description of the PR. This causes GitHub to close it automatically when the PR is merged.

This is documentation update for https://github.com/iterative/dvc/pull/4622.

Please choose to [allow us](https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/allowing-changes-to-a-pull-request-branch-created-from-a-fork) to edit your branch when creating the PR.

Thank you for the contribution - we'll try to review it as soon as possible. 🙏

### CODE REVIEW

**Review by jorgeorpinel — APPROVED** [review]
Thank you @DiPaolo 
Can you please try to let us know when iterative/dvc/pull/4622 is merged, to do the same here after that?
Thanks

**Comment by efiop** [comment]
@jorgeorpinel  iterative/dvc/pull/4622 is merged :slightly_smiling_face:

File: content/docs/command-reference/import.md [hunk 1]
```diff
@@ -86,6 +88,11 @@ repo at `url`) are not supported.
   current working directory). Directories specified in the path must already
   exist, otherwise this command will fail.
 
+- `--file <filename>` - specify a path and/or file name for the `.dvc` file
+  created by this command (e.g. `--file stages/stage.dvc`). This overrides the
```
Comment (shcheklein) [inline content/docs/command-reference/import.md:92]: better to avoid using stage in the import context (for the future cc @jorgeorpinel )

Comment (jorgeorpinel) [reply content/docs/command-reference/import.md:92]: Agree. We have a pending check box about this in #608 epic. Also rel=#1366

**Pull request merged**